tests/cli/test_sysctl_init.py

```diff
@@ -195,6 +197,6 @@ def register_signal_handlers(self) -> None:
         self.assertTrue(run_node is not None)
         conn = run_node.manager.connections
 
-        curr_always_enabled_sync = list(conn.always_enable_sync)
+        curr_always_enabled_sync = list(map(str, conn.always_enable_sync))
```

Comment:
Why this change?

Reply:
I had to make this for the test to work, because the assertion bellow needs sets of the same type, and `conn.always_enable_sync` is a `set[PeerId]`, but now that you mention it I think this is only changed in the next PR, so I might have backported this fix too early.

Reply:
Ok!